pynest/examples/eprop_plasticity/README.rst

```diff
@@ -0,0 +1,26 @@
+E-prop plasticity examples
+==========================
+
+
+.. image:: ../../../../pynest/examples/eprop_plasticity/eprop_supervised_regression_infrastructure.png
+
+Eligibility propagation (e-prop) [1]_ is a three-factor learning rule
+for spiking neural networks that approximates backpropagation through time.
+The original TensorFlow implementation of e-prop was demonstrated, among others, on a supervised regression task to generate temporal patterns and a supervised
+classification task  to accumulate evidence [2]_.  Here, you find tutorials on how to reproduce these two tasks using the NEST
+implementation of e-prop [3]_ and how to visualize the simulation
+recordings.
+
```

Comment:
Could you add links to the specific examples here
```suggestion
* :doc:`/auto_examples/eprop_plasticity/eprop_supervised_regression`
* :doc:`/auto_examples/eprop_plasticity/eprop_supervised_classification`
```
They are built and placed automatically by sphinx-gallery at the bottom of the page, but I think it would be more obvious  closer to the top

Reply:
Yes, they are placed prominently at the end of the page. Wouldn't it be confusing for the user if there were two set of links given in the page that lead to the same destinations? If two sets of links are given, I would assume that they lead to different destinations, since otherwise they would be redundant. Since the text is not very long immediately after the references there are two big clickable buttons with the descriptions "Supervised regression task" and "Supervised classification task". I would prefer to leave it like this for e-prop. In the cortical microcircuit model it's a different case, since the text there is very long and the one needs to scroll quite a bit to get to the clickable buttons - so there it makes sense to have additional links closer to the top.

Reply:
ok, we can leave it